daemon/api_snaps.go

```diff
@@ -145,6 +145,7 @@ func postSnap(c *Command, r *http.Request, user *auth.UserState) Response {
 		return BadRequest("unknown action %s", inst.Action)
 	}
 
+	// TODO use an epxlicit context parameter
```

Comment:
typo

Reply:
thanks for catching, I've updated the patch